tools/make/helm.mk

```diff
@@ -5,7 +5,7 @@
 include tools/make/env.mk
 
 
-OCI_REGISTRY ?= oci://${REGISTRY}
+OCI_REGISTRY ?= oci://${REGISTRY}/linuxsuren
```

Comment:
We missed here @yuluo-yx

Reply:
no, It defined in env.mk 

https://github.com/LinuxSuRen/api-testing/blob/9f400bbcb9b3923a26232f90eb6b11eaa5ea9896/tools/make/env.mk#L10

Reply:
It looks like the makefile is not written perfectly, if there are any other problems, please let me know and I'll fix it as fast as I can! 🫡

Reply:
But it was failed.

https://github.com/LinuxSuRen/api-testing/actions/runs/9012966626/job/24762996674

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/aeecfc52-593d-4424-9c9d-a40389a5c9eb)

Reply:
I figured out the reason. See https://github.com/LinuxSuRen/api-testing/blob/9f400bbcb9b3923a26232f90eb6b11eaa5ea9896/.github/workflows/release.yaml#L11

Actually, `docker.io/linuxsuren` is not a registry. Normally a registry should be `docker.io`.

Reply:
linuxsuren is similar to namespance, docker.io is registry address . maybe this?

Reply:
I will fixed it.

Reply:
> linuxsuren is similar to namespance, docker.io is registry address . maybe this?

Agree.

Reply:
fixed, in https://github.com/LinuxSuRen/api-testing/pull/424